hasura/migrations/default/1687894707285_run_sql_migration/down.sql

```diff
@@ -0,0 +1,10 @@
+DROP VIEW "public"."contribution_count" AS 
+ SELECT p.id AS profile_id,
```

Comment:
we don't need this whole AS SELECT query part. Just DROP VIEW "public"."contribution_count"